jsconfig.json

```diff
@@ -0,0 +1,7 @@
+
+{
+  "compilerOptions": {
+    "baseUrl": "src"
+  },
+  "include": ["src"]
+}
```

Comment:
do we need a newline at this end of this?

Reply:
Not sure, but I think a new line at the end of JSON files is not a necessity.
Though that new line at the beginning of the file should not be there lol